cpp/ps0005/main.cpp

```diff
@@ -3,41 +3,44 @@ using namespace std;
 
 int main()
 {
-    float A;
-    cout << "Please enter A ";
-    cin >> A;
+    // prompting user for the 2 operands and an operator 2/2
+    float num1;
+    cout << "Please enter number 1: ";
+    cin >> num1;
 
-    float B;
-    cout << "Please enter B ";
-    cin >> B;
+    float num2;
+    cout << "Please enter number 2: ";
+    cin >> num2;
 
     char op;
     cout << "Please enter an operator ";
     cin >> op;
 
+    // checking if the operator is valid 4/4
     if (op == '+')
-        cout << "A + B = " << A + B << endl;
+        cout << num1 << " + "  << num2 << " = " << num1 + num2 << endl;
     else if (op == '-')
-        cout << "A - B = " << A - B << endl;
+        cout << num1 << " - "  << num2 << " = " << num1 - num2 << endl;
     else if (op == '*')
-        cout << "A * B = " << A * B << endl;
+        cout << num1 << " * "  << num2 << " = " << num1 * num2 << endl;
     else if (op == '/')
     {
-        if (B != 0)
-            cout << "A / B = " << A / B << endl;
+        if (num2 != 0)
+            cout << num1 << " / "  << num2 << " = " << num1 / num2 << endl;
         else
             cout << "Division by zero is not allowed." << endl;
-    }
+    } // 1/1
     else if (op == '%')
     {
-
-        if (B != 0 && A == (int)A && B == (int)B)
-            cout << "A % B = " << (int)A % (int)B << endl;
+        if (num2 != 0 && num1 == (int)num1 && num2 == (int)num2)
+            cout << num1 << " % "  << num2 << " = " << (int)num1 % (int)num2 << endl;
         else
-            cout << " A and B must be integers for modulus operation, and B must not be zero." << endl;
-    }
+            cout << " number 1 and number 2 must be integers for modulus operation, and number 2 must not be zero." << endl;
+    } // 1/1
     else
-        cout << "Invalid operator!" << endl;
-
+        cout << "Invalid operator!" << endl; // 1/1
+    // 1/1

```

Comment:
- [x] used `switch` and `if` structures in a right way to handle different cases
- [x] handled division operation `/`
- [x] handled modulo operation `%`
- [x] handled invalid operator case
- [x] chose the right operation for each case